pkgs/by-name/lm/lm_sensors/package.nix

```diff
@@ -80,28 +63,24 @@ stdenv.mkDerivation {
     "MANDIR=${placeholder "man"}/share/man"
     # This is a dependency of the library.
     "ETCDIR=${placeholder "out"}/etc"
+    "BUILD_SHARED_LIB=${if stdenv.hostPlatform.isStatic then "0" else "1"}"
+    "BUILD_STATIC_LIB=${if stdenv.hostPlatform.isStatic then "1" else "0"}"
 
     "CC=${stdenv.cc.targetPrefix}cc"
     "AR=${stdenv.cc.targetPrefix}ar"
   ] ++ lib.optional sensord "PROG_EXTRA=sensord";
 
   enableParallelBuilding = true;
 
-  # Making regexp to patch-out installing of .so symlinks from Makefile is
-  # complicated, it is easier to remove them post-install.
-  postInstall =
-    ''
-      mkdir -p $doc/share/doc/lm_sensors
-      cp -r configs doc/* $doc/share/doc/lm_sensors
-    ''
-    + lib.optionalString stdenv.hostPlatform.isStatic ''
-      rm $out/lib/*.so*
-    '';
+  postInstall = ''
+    mkdir -p $doc/share/doc/lm_sensors
+    cp -r configs doc/* $doc/share/doc/lm_sensors
+  '';
 
   meta = {
     homepage = "https://hwmon.wiki.kernel.org/lm_sensors";
-    changelog = "https://raw.githubusercontent.com/lm-sensors/lm-sensors/${tag}/CHANGES";
-    description = "Tools for reading hardware sensors";
+    changelog = "https://raw.githubusercontent.com/hramrach/lm-sensors/${tag}/CHANGES";
+    description = "Tools for reading hardware sensors - openSUSE fork";
```

Comment:
Is it okay to call it "openSUSE fork"? I'm not sure if @hramrach is doing it on behalf of OpenSUSE, or as a personal fork that is used by distros.

Reply:
My understanding is that it is in his capacity as an opensuse team member, but we can of course also just drop that bit here. We don't really gain anything by leaving it in here.